Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1636 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1636 |
| Installation process | http://install.openy-dev.ffwua.com/build1636/install.php |

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1637/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #583 of commit 3d4532b4211511652d2b72bba9b6a20652143b1c automatically merged.
GitHub pull request #583 of commit 3d4532b4211511652d2b72bba9b6a20652143b1c automatically merged.
GitHub pull request #583 of commit 3d4532b4211511652d2b72bba9b6a20652143b1c automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git9217829586334340906.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git4609265106888419345.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/583/merge^{commit} # timeout=10
 > git branch -a --contains 26098d5d9c3ddc09972b6156a468736299ff41f4 # timeout=10
 > git rev-parse remotes/origin/pr/583/merge^{commit} # timeout=10
Checking out Revision 26098d5d9c3ddc09972b6156a468736299ff41f4 (origin/pr/583/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 26098d5d9c3ddc09972b6156a468736299ff41f4
FATAL: Could not checkout 26098d5d9c3ddc09972b6156a468736299ff41f4
hudson.plugins.git.GitException: Could not checkout 26098d5d9c3ddc09972b6156a468736299ff41f4
	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl$8.execute(CliGitAPIImpl.java:1867)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1060)
	at hudson.scm.SCM.checkout(SCM.java:485)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1277)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:610)
	at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:86)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:532)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
Caused by: hudson.plugins.git.GitException: Command "git checkout -f 26098d5d9c3ddc09972b6156a468736299ff41f4" returned status code 128:
stdout: 
stderr: error: unable to write file modules/openy_features/openy_demo_content/modules/openy_demo_menu/modules/openy_demo_menu_main/assets/images/health_and_fitness.png
error: unable to write file modules/openy_features/openy_demo_content/modules/openy_demo_menu/modules/openy_demo_menu_main/assets/images/kids_and_family_activities.png
error: unable to write file modules/openy_features/openy_demo_content/modules/openy_demo_menu/modules/openy_demo_menu_main/assets/images/swimming.png
error: unable to write file modules/openy_features/openy_demo_content/modules/openy_demo_menu/modules/openy_demo_menu_main/assets/images/youth_programs.png
fatal: cannot create directory 
```

--- a/behat.yml
+++ b/behat.yml
@@ -67,13 +67,5 @@ default:
       print_args: true
       print_outp: true
       loop_break: true
-    Cevou\Behat\ScreenshotCompareExtension:
-      screenshot_dir: %paths.base%/features/screenshots
-      sessions:
-        default: ~
-      adapters:
-        default:
-          local:
-            directory: %paths.base%/error_screenshots
 imports:
   - behat.local.yml

--- a/behat.yml
+++ b/behat.yml
@@ -12,7 +12,6 @@ default:
         - Drupal\DrupalExtension\Context\MinkContext
         - Drupal\DrupalExtension\Context\MessageContext
         - Drupal\DrupalExtension\Context\DrushContext
-        - Cevou\Behat\ScreenshotCompareExtension\Context\ScreenshotCompareContext
   formatters:
     html:
       output_path: %paths.base%/build/reports/behat

